composer.json

```diff
@@ -15,7 +14,7 @@
         "symfony/translation": "~2.3",
         "pagerfanta/pagerfanta": "dev-master",
         "franmomu/silex-pagerfanta-provider": "dev-master",
-        "doctrine/dbal": "2.2.*",
+        "doctrine/dbal": "v2.5.1",
```

Comment:
Why are you bumping the dbal version, why are you requiring `2.5.1` specifically instead of `~2.5`, and why did you add a `v` while all the others don't use it?